src/AddressRegistry.Api.BackOffice.Handlers.Lambda/Function.cs

```diff
@@ -1,25 +1,24 @@
 using Amazon.Lambda.Core;
+using Amazon.Lambda.Serialization.Json;
 
-[assembly: LambdaSerializer(typeof(Amazon.Lambda.Serialization.Json.JsonSerializer))]
+[assembly: LambdaSerializer(typeof(JsonSerializer))]
```

Comment:
for clarity please leave the entire namespace in this line

Reply:
I've put it back. Rider shortcut changed it.